index.html

```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/3085842107.js" crossorigin="anonymous"></script>
+    <script defer src="/node_modules/bootstrap/dist/js/bootstrap.bundle.js"></script>
+    <link rel="stylesheet" href="/dist/css/main.css">
+    <title>css frameworks</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <div class="container">
+                <div class="row justify-content-center">
+                    <div class="col">
+                        <img src="assets/images/logo.png" alt="image of the phillers logo">
+                    </div>
+                    <hr>
+                    <div class="row mb-4">
+                      <div class="col-12">
+                        <nav class="nav align-items-center">
+                            <ul class="list-inline col-12 col-md-8 align-self-start d-flex justify-content-start">
+                                <li class="list-inline-item col-md-3 col-sm-4 col-4"><a href="/" class="nav-link p-0 text-active text-decoration-underline"> Home</a></li>
+                                <li class="list-inline-item col-md-3 col-sm-4 col-4"><a href="/login.html" class="nav-link p-0">Log in</a></li>
+                                <li class="list-inline-item col-md-3 col-sm-4 col-4"><a href="/profile.html" class="nav-link p-0"> Profile</a></li>
+                            </ul>
+                            <div class="input-group col">
+                                <input type="search" id="form1" class="form-control col-1" />
+                                <button type="button" class="btn btn-primary">
+                                    <span class="fas fa-search"></span>
+                                </button>
+                            </div>
+                          </div>
+                        </nav>  
+                    </div>
+                </div>
+            </div>
+        </header>
+        <main>
+            <section class="user-posts">
+                <div class="container m-0">
```

Comment:
```suggestion
                <div class="container m-0">
```

This sticks out a bit. One of the purposes of a container is to use margin auto to horizontally align the page. If you don't want this you should override the whole container class instead so it's consistent.

Reply:
I kinda wanted it to stick to the wrappers max width

Reply:
Then you could use `.container-fluid`

Reply:
Cool. I'll check em both out and see what fits the best! Thanks man